plugin.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
+        id="edu.berkeley.eecs.emission.cordova.integritydetect"
+        version="1.0.0">
+
+  <name>IntegrityDetection</name>
+  <description>Integrity detection for road surface </description>
+  <author>Bingbing Huang (bing2013@cs.washington.edu)</author>
+  <license>Modified BSD license</license>
+
+  <engines>
+    <engine name="cordova" version=">=3.6.0"/>
+  </engines>
+
+  <platform name="android">
+    <config-file target="res/xml/config.xml" parent="/*">
+      <feature name="integrityDetect">
+        <param name="android-package" value="edu.berkeley.eecs.emission.cordova.integrityDetect.IntegrityNotificationReceiver"/>
+      </feature>
+    </config-file>
```

Comment:
you also need to add a broadcast receiver to config.xml through `plugin.xml`. See https://github.com/e-mission/e-mission-transition-notify/blob/master/plugin.xml#L43